### PR TITLE
Erroneous mimetypes invalidated, not thrown

### PR DESCRIPTION
**Pull request by why-not-try-calmer**

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2490 When QFieldCloud does not recognize the mimetype of the uploaded file, just invalidate the field, do not crash](https://app.clickup.com/t/2192114/QF-2490)

**Comment by why-not-try-calmer** [comment]
The same idea could be used for `upload_project_thumbail`.

File: docker-app/qfieldcloud/core/utils2/storage.py [hunk 1]
```diff
@@ -262,8 +263,14 @@ def file_response(
     )
 
 
+class Mimes(str, Enum):
```
Comment (suricactus) [inline docker-app/qfieldcloud/core/utils2/storage.py:266]: Some more descriptive name than `Mimes` would help a lot. Like `AvatarMimeTypes` or `ImageMimeTypes` or else?

**Comment by why-not-try-calmer** [comment]
I am no able to reproduce the CI error locally. Tests pass A-OK here.

**Comment by miili** [comment]
For what it's worth: Use `Literal` instead.

```py
ImageMimeType = Literal["image/svg+xml", "image/png", "image/jpeg"]
```

**Comment by why-not-try-calmer** [comment]
> For what it's worth: Use `Literal` instead.
> 
> ```python
> ImageMimeType = Literal["image/svg+xml", "image/png", "image/jpeg"]
> ```

Enums are fine. The name and the value are distinct, and consumed differently.

**Review by suricactus — APPROVED** [review]

**Pull request merged**